cmds/contd/go.sum

```diff
@@ -69,6 +69,7 @@ github.com/google/shlex v0.0.0-20181106134648-c34317bd91bf/go.mod h1:RpwtwJQFrIE
 github.com/google/uuid v1.1.1 h1:Gkbcsh/GbpXz7lPftLA3P6TYMwjCLYm83jiFQZF/3gY=
 github.com/google/uuid v1.1.1/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
+github.com/iancoleman/strcase v0.0.0-20190422225806-e506e3ef7365/go.mod h1:SK73tn/9oHe+/Y0h39VT4UCxmurVJkR5NA7kMEAOgSE=
```

Comment:
Why is this added ? I don't see any new dependency being added in this PR.

Reply:
Hmm, weird it's from the schema actually, I don't know how it got here!

Reply:
Ok no worries, we can merge the schema PR first, rebase this branch, so these things will disappear

Reply:
This doesn't go with rebase. I think we can ignore it since it's not a change to the `go.toml` so this is basically added by `go build`